pom.xml

```diff
@@ -91,7 +93,7 @@
       <dependency>
         <groupId>com.google.guava</groupId>
         <artifactId>guava</artifactId>
-        <version>14.0.1</version>
+        <version>31.0.1-jre</version>
```

Comment:
Because of this I seem to be running into https://issues.sonatype.org/browse/NEXUS-23594
```
[INFO] Uploaded to ossrh: https://s01.oss.sonatype.org:443/service/local/staging/deployByRepositoryId/iogithubgitflow-incremental-builder-1000/io/github/gitflow-incremental-builder/gitflow-incremental-builder/4.0.0/gitflow-incremental-builder-4.0.0-sources.jar.asc (499 B at 2.0 kB/s)
[INFO] [INFO]  * Upload of locally staged artifacts finished.
[INFO] [INFO]  * Closing staging repository with ID "iogithubgitflow-incremental-builder-1000".
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] BUILD FAILURE
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] Total time:  01:54 min
[INFO] [INFO] Finished at: 2022-02-21T00:43:24+01:00
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [ERROR] Failed to execute goal org.sonatype.plugins:nexus-staging-maven-plugin:1.6.11:deploy (injected-nexus-deploy) on project gitflow-incremental-builder: Execution injected-nexus-deploy of goal org.sonatype.plugins:nexus-staging-maven-plugin:1.6.11:deploy failed: An API incompatibility was encountered while executing org.sonatype.plugins:nexus-staging-maven-plugin:1.6.11:deploy: java.lang.IllegalAccessError: class com.sonatype.nexus.staging.client.internal.StagingWorkflowV3ServiceImpl tried to access method 'void com.google.common.base.Stopwatch.<init>()' (com.sonatype.nexus.staging.client.internal.StagingWorkflowV3ServiceImpl and com.google.common.base.Stopwatch are in unnamed module of loader org.codehaus.plexus.classworlds.realm.ClassRealm @64030b91)
[INFO] [ERROR] -----------------------------------------------------
```

Reply:
@famod please try release 1.6.12, I believe the issue is addressed in that release.